### PR TITLE
Welcome

### PR DESCRIPTION
**Pull request by cyulater123**
Sorry if something is really awkward --- this is a really big adventure for me through Github.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import gci2 from './gci2.png';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={gci2} className="App-logo" alt="logo" />
```
Comment (f4ww4z) [inline App.js]: The image seems to be titled. Remove the animation associated with it.

Comment (cyulater123) [reply App.js]: Hey! After uploading package.json, I realized I couldn't load the website, so I ended up deleting it. I'll reupload it now.

Comment (cyulater123) [reply App.js]: > The image seems to be titled. Remove the animation associated with it.

I ended up editing the App.css file to remove the animation.

**Comment by cyulater123** [comment]
> Approving this @cyulater123 , but please fix the merge conflict.

Hey f4ww4z!
Merge conflict has been resolved.

**Pull request merged**